CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ Changelog
 =========
 
 ## [Unreleased]
+* PRIVATE-VLAN
```

Comment:
It's in the right section now, but I meant for you to put it under the Added section.  Sorry for the confusion! :smiley: 

https://github.com/davidefdl/cisco-network-node-utils/blob/feature/private-vlan/CHANGELOG.md#added